skills/implementation-logger/SKILL.md

```
@@ -0,0 +1,394 @@
+
+- **Before Implementation**: Blind spot pass + spec interview reduce unknowns
+- **During Implementation**: Use this skill to track deviations
+- **After Implementation**: Extract for PR description, ADRs, MEMORY.md
+- **Follow-up**: Quiz-me to verify you understand the logged decisions
+
+## Output Locations
+
+### Where to Store Logs
+
+**Temporary logs** (during work):
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)